tests/src/Search/Web/ElasticTestCase.php

```diff
@@ -139,6 +139,7 @@ public function getJsonResponse()
     {
         $response = $this->getResponse();
         if (!$response->isOk()) {
+            var_dump($response);
```

Comment:
Ah sorry forgot to remove that, I only used that for debugging. I've moved it into the `fail(...)`